tutorials/01_introductory/sample_plots.py

```diff
@@ -58,8 +59,8 @@
 You can add arbitrary paths in matplotlib using the
```

Comment:
not sure - I didn't create any of this text I just moved it here :-P  I'll call it "other paths" I guess